### PR TITLE
getenforce: add page

### PR DESCRIPTION
**Pull request by jimmychang851129**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by jimmychang851129** [comment]
This PR links to https://github.com/tldr-pages/tldr/issues/11896

File: pages/linux/getenforce.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# getenforce
+
+> Get the current mode of SELinux (i.e. enforcing, permissive, or disabled).
+> More information: <https://man7.org/linux/man-pages/man8/getenforce.8.html>.
```
Comment (CleanMachine1) [inline pages/linux/getenforce.md:4]:
```suggestion
> More information: <https://manned.org/man/getenforce>.
```

File: pages/linux/getenforce.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# getenforce
+
+> Get the current mode of SELinux (i.e. enforcing, permissive, or disabled).
```
Comment (CleanMachine1) [inline pages/linux/getenforce.md:3]:
```suggestion
> Get the current mode of SELinux (e.g. enforcing, permissive, or disabled).
```

Comment (CleanMachine1) [reply pages/linux/getenforce.md:3]: ie is `that is`
eg is `for example`

**Review by CleanMachine1 — COMMENTED** [review]
Thanks for your contribution.

File: pages/linux/getenforce.md (hunk 2 above)
Comment (gutjuri) [reply pages/linux/getenforce.md:3]: I don't think that "i.e." is wrong here (if the provided list of possible modes is exhaustive)

**Review by gutjuri — APPROVED** [review]

File: pages/linux/getenforce.md (hunk 2 above)
Comment (cyqsimon) [reply pages/linux/getenforce.md:3]: `i.e.` is correct here - the 3 modes are exhaustive.

**Review by cyqsimon — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**